src/pages/[platform]/build-a-backend/data/connect-event-api/index.mdx

```diff
@@ -89,15 +89,182 @@ export default function App() {
+import { Policy, PolicyStatement } from 'aws-cdk-lib/aws-iam';
+// highlight-end
+
+const backend = defineBackend({
+	auth,
+});
+
+// highlight-start
+// create a new stack for our Event API resources:
+const customResources = backend.createStack('custom-resources');
+
+// add a new Event API to the stack:
+const cfnEventAPI = new CfnApi(customResources, 'cfnEventAPI', {
+  name: 'my-event-api',
+  eventConfig: {
+    authProviders: [
+      {
+        authType: AuthorizationType.USER_POOL,
+        cognitoConfig: {
+          awsRegion: customResources.region,
+          // configure Event API to use the Cognito User Pool provisioned by Amplify:
+          userPoolId: backend.auth.resources.userPool.userPoolId,
+        },
+      },
+    ],
+    // configure the User Pool as the auth provider for Connect, Publish, and Subscribe operations:
+    connectionAuthModes: [{ authType: AuthorizationType.USER_POOL }],
+    defaultPublishAuthModes: [{ authType: AuthorizationType.USER_POOL }],
+    defaultSubscribeAuthModes: [{ authType: AuthorizationType.USER_POOL }],
+  },
+});
+
+// create a default namespace for our Event API:
+const namespace = new CfnChannelNamespace(
+  customResources,
+  'cfnEventAPINamespace',
+  {
+    apiId: cfnEventAPI.attrApiId,
+    name: 'default',
+  }
+);
+
+// attach a policy to the authenticated user role in our User Pool to grant access to the Event API:
+backend.auth.resources.authenticatedUserIamRole.attachInlinePolicy(
+  new Policy(customResources, 'AppSyncEventPolicy', {
+    statements: [
+      new PolicyStatement({
+        actions: [
+          'appsync:EventConnect',
+          'appsync:EventSubscribe',
+          'appsync:EventPublish',
+        ],
+        resources: [`${cfnEventAPI.attrApiArn}/*`, `${cfnEventAPI.attrApiArn}`],
+      }),
+    ],
+  })
+);
+
+// finally, add the Event API configuration to amplify_outputs:
+backend.addOutput({
+  custom: {
+    events: {
+      url: `https://${cfnEventAPI.getAtt('Dns.Http').toString()}/event`,
+      aws_region: customResources.region,
+      default_authorization_type: AuthorizationType.USER_POOL,
+    },
+  },
+});
+// highlight-end
+```
+
+### Deploy Backend
+
+To test your changes, deploy your Amplify Sandbox.
+
+```bash title="Terminal" showLineNumbers={false}
+npx ampx sandbox
+```
+
+### Connect your frontend application
+
+After the sandbox deploys, connect your frontend application to the Event API.
+We'll be using the Amplify Authenticator to sign in to our Cognito User Pool.
+
+If you don't already have the Authenticator installed, you can install it by running `npm i @aws-amplify/ui-react`.
+
+```tsx title="src/App.tsx"
+import { useEffect, useState } from 'react';
+import { Amplify } from 'aws-amplify';
+import { events, type EventsChannel } from 'aws-amplify/data';
+import { Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+import config from '@/amplify_outputs.json';
+
+Amplify.configure(config);
```

Comment:
```suggestion
import outputs from '@/amplify_outputs.json';

Amplify.configure(outputs);
```

Reply:
minor tweak to use "outputs" language, and to remove any confusion for Vite-based applications we can use a relative import. The `@` alias is available out of the box for Next.js apps, but customers would need to configure this for Vite